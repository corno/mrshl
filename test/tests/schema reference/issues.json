[
    [ "expected '(' but found '{'", "warning", 2, 1, 2, 2],
    [ "unexpected property: 'a'", "error", 3, 5, 3, 8],
    [ "expected ')' but found '}'", "warning", 6, 1, 6, 2],
    [ "missing property: 'a1'", "error", 2, 1, 2, 2]
]