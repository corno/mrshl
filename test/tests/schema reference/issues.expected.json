[
    [ "deserialization", "warning", "expected a verbose type: ( )", 2, 1, 2, 2],
    [ "deserialization", "warning", "expected a dictionary: { }", 3, 25, 3, 26],
    [ "deserialization", "error", "unexpected property: 'a'. Choose from 'component types', 'root type'", 6, 5, 6, 8],
    [ "deserialization", "error", "unexpected property: 'b'. Choose from 'component types', 'root type'", 7, 5, 7, 8],
    [ "deserialization", "error", "property 'component types' has default value, remove", 3, 5, 3, 22]
]